tsconfig.repo-config-files.json

```
@@ -4,22 +4,29 @@
     "outDir": "./dist",
     "allowJs": true,
     "allowImportingTsExtensions": true,
-    "erasableSyntaxOnly": true
+    "erasableSyntaxOnly": true,
+    "paths": {
+      "@site/*": ["./packages/website/*"]
+    }
```

Why is this in the root `tsconfig` instead of `website`'s?

```suggestion
    "paths": {
      "@site/*": ["./packages/website/*"]
    }
```